### DISCUSSION TITLE
CopyArchiveToContainerCmd to accept ownership params for the file

### DISCUSSION THREAD

**Discussion by mhdrafic** [docker-java/docker-java#1926] [Q&A]
Is there a way we can specify the ownership of the file that is being copied to the container.

Currently when we copy the file with "withHostResource" option, the file is copied to root directory of the container ("/") and file ownership is also "root".

The requirement is like, we need to copy it to the same folder as mentioned above but the file ownership should be configurable.

**Comment by k-wall** [comment]
Will https://github.com/docker-java/docker-java/pull/1963 help you?
